WAW.API/Auth/Resources/UserRequest.cs

```diff
@@ -1,23 +1,32 @@
 using System.ComponentModel.DataAnnotations;
+using Swashbuckle.AspNetCore.Annotations;
 
 namespace WAW.API.Auth.Resources;
 
+[SwaggerSchema(Required = new[]{"FullName", "PreferredName", "Email", "Birthdate", "Location",})]
```

Comment:
Isn't this redundant since we are already using `[Required]` annotations?

Check if this is needed, else remove it from here and other schemas that have it.